README.rst

```diff
@@ -245,6 +245,32 @@ This function can be overridden, see Configuration_.
     }
   );
 
+References
+----------
+
+References are useful if you want to open a showroom item with some additional links out of a text or something.
```

Comment:
with some additional links out of a text or something -> that is referenced by more than one element on the same page.